src/tests/Dashboard/components/DictionariesPaginations.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import expect from 'expect';
+
+import Paginations from '../../../components/dashboard/components/DictionariesPaginations';
+
+describe('Test suite for dictionaries pagination component', () => {
+  it('Should render without breaking', () => {
+    const props = {
+      dictionaries: 10,
+      firstDictionaryIndex: 1,
+      totalDictionaries: 20,
+      currentPage: 1,
+      handleClick: jest.fn(),
+      lastPage: 2,
+      onChangeLimit: jest.fn(),
+    };
+    const wrapper = mount(<Paginations {...props} />);
+    const selectLimit = wrapper.find('select');
+    selectLimit.simulate('change', { target: { value: 25 } });
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('Should go next page', () => {
+    const props = {
+      dictionaries: 10,
+      firstDictionaryIndex: 1,
+      totalDictionaries: 20,
+      currentPage: 1,
+      handleClick: jest.fn(),
+      lastPage: 2,
+      onChangeLimit: jest.fn(),
+    };
+    const wrapper = mount(<Paginations {...props} />);
+    wrapper.find('.nxt').simulate('click');
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('Should go to the previous page', () => {
```

Comment:
Same here.

Reply:
@emmabaye Thanks for this. Let correct this.